src/Resources/Pages/UserDetails/UserDetails.jsx

```diff
@@ -0,0 +1,59 @@
+//Impor from react
+import { useEffect } from 'react';
+
+//Import atoms
+import { userDetails } from "../../../RecoilResources/Atoms";
+
+//Import from recoil
+import { useRecoilState } from "recoil";
+
+//Import from react-bootstrap
+import { Image } from 'react-bootstrap';
+
+//Import images
+import facebookIcon  from '../../../Assets/facebook-icon.svg';
+import gmailIcon from '../../../Assets/gmail-icon.svg';
+
+//Import from fa
+import { faPhone } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const UserDetails = (props) => {
+
+    const [displaiedUserDetails] = useRecoilState(userDetails);
+
+    // This hook equales componentWillUnmount lifecycle function
+    useEffect(() => {
+        return () => {
+            window.location.reload();
+        }
+    }, [])
+
+    console.log(displaiedUserDetails);
+    
+    return (
+        <div style={{marginTop: '8rem', marginBottom: '5rem' ,display: 'flex', flexDirection: 'column', alignItems: 'center'}}>
+            <Image style={{width: '30%', minWidth: '280px' }} src={displaiedUserDetails.photo.url}/>
```

Comment:
displayedUserDetails typo and in subsequent lines

Reply:
Oops..